src/preview.ts

```diff
@@ -1,4 +1,4 @@
-import type { Preview } from "@storybook/react";
+import type { Preview } from "@storybook/react-vite";
 import { withApolloClient } from "./withApolloClient";
 
 const preview: Preview = {
```

Comment:
@yannbf @shilman I'm guessing this is no longer the right way to do things because other people might use a different bundler.